dependencies.yaml

```diff
@@ -466,45 +459,15 @@ dependencies:
             packages:
               - --extra-index-url=https://download.pytorch.org/whl/cu118
           - {matrix: null, packages: null}
-      - output_types: [requirements, pyproject]
-        matrices:
-          - matrix: {cuda: "12.*"}
-            packages:
-              - &pytorch_pip torch>=2.3
-              - *tensordict
-          - matrix: {cuda: "11.*"}
-            packages:
-              - *pytorch_pip
-              - *tensordict
-          - {matrix: null, packages: [*pytorch_pip, *tensordict]}
-
-  # Will remove this after snap-stanford/ogb#497 is resolved.
-  # Temporarily sets the max pytorch version to 2.5 for compatibility
-  # with ogb.
-  depends_on_ogb:
-    common:
       - output_types: [conda]
-        packages:
-          - pytorch>=2.3,<2.6a0
-    specific:
-      - output_types: [requirements]
         matrices:
-          - matrix: {cuda: "12.*"}
-            packages:
-              - --extra-index-url=https://download.pytorch.org/whl/cu121
```

Comment:
Just want to mention that torch 2.6 moves away from CUDA 12.1. We should use 12.4 or 12.6 if needed.

Reply:
Will handle in a separate PR.

Reply:
https://github.com/rapidsai/cugraph-gnn/issues/154